### DISCUSSION TITLE
How to highlight the xml file?

### DISCUSSION THREAD

**Discussion by ws1922667298** [nvim-treesitter/nvim-treesitter#4519] [Q&A] [answered]
hello! 
As title I don't know how to set treesitter plugin for highlight xml file.
I'm going to use nvim to develop java project, when setting up the pom.xml file encountered this problem.
thank for your answer.

**Comment by amaanq** [comment, marked as answer]
you can set the ft to html and use that parser, we don't have a dedicated xml one for now.

**Reply by ws1922667298** [reply]
yes, thank you!Let me explore how to make it default to HTML.

**Reply by ws1922667298** [reply]
great.I create `ftplugin/xml.lua` and add `vim.opt.filetype = "html"` now it can be used normally.
Thank.
